packages/client/hmi-client/src/services/charts.ts

```diff
@@ -370,7 +370,7 @@ export function createOptimizeChart(
 						field: 'tag',
 						type: 'nominal',
 						scale: {
-							domain: ['out', 'in'],
+							domain: ['failing', 'passing'],
```

Comment:
I am not sure if you can mess around with the domain like that, because `in` and `out` have some special meanings in @blanchco 's logic if I recall

Reply:
hm.. I think you are right. I'll take a look at if there are other ways.

Reply:
@jryu01 you may need to use something like this in the transform array:

```
{
        calculate: "datum.tag === 'out' ? 'failing' : 'passing'", // Add this line
        as: 'tagLabel' // Use a new field name to store the mapped value
}
```

and then use tagLabel as the new field name